### ISSUE TITLE
when column and data rows is big the silce explor js will allway crash

### ISSUE THREAD

**Issue by qinzl1** [apache/incubator-superset#2706]
Make sure these boxes are checked before submitting your issue - thank you!

- [ ] I have checked the superset logs for python stacktraces and included it here as text if any
- [ ] I have reproduced the issue with at least the latest released version of superset
- [ ] I have checked the issue tracker for the same issue and I haven't found one similar

**Comment by xrmx** [comment]
Please define crash and define big.

**Comment by qinzl1** [comment]
@xrmx who can i fand the config

**Comment by xrmx** [comment]
@qinzl1 i mean to define in plain english. What is a crash for you? Where does it happen?How much data is big?

**Comment by mistercrunch** [comment]
you can lower `ROW_LIMIT` in your `superset_config.py`

**Comment by qinzl1** [comment]
@xrmx 8 fields and 3000 rows. Visualization Type is table view, anther problem is the table view have no Drag scroll bar

**Comment by qinzl1** [comment]
@xrmx if then silce explore  table view Visualization can paging

**Comment by qinzl1** [comment]
if i set 'Page Length' ,the table view can not display all row. it only return limit  100 row

**Comment by mistercrunch** [comment]
Notice: this issue has been closed because it has been inactive for 333 days. Feel free to comment and request for this issue to be reopened.
